.github/workflows/ci-pytorch_test-conda.yml

```diff
@@ -24,11 +24,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8"]  # previous to last Python version as that one is already used in test-full
-        pytorch-version: ["1.9", "1.10"]
         # nightly: add when there's a release candidate
         include:
+          - {python-version: "3.7", pytorch-version: "1.9"}
```

Comment:
why python version is changed for PT-1.9?

Reply:
just diversification, not a specific reason...
there is still a high number of users with py3.7